.drone.yml

```diff
@@ -668,43 +668,49 @@ trigger:
   event:
     - promote
   target:
-    # deploy the dataprotector subgraph
-    - subgraph-deploy-dev
     # deploy the dataprotector subgraph for the staging environment
     - subgraph-deploy-staging
+    # deploy the dataprotector subgraph
+    - subgraph-deploy-tmp
     # deploy the dataprotector subgraph for the prod environment
     - subgraph-deploy-prod
   branch:
     - develop
     - main
 
 steps:
-  - name: deploy-v2-dev
+  - name: install deps
+    image: node:18.19
+    pull: always
+    commands:
+      - cd packages/subgraph
+      - npm ci
+
+  - name: deploy-v2-staging
     image: node:18.19
     params:
       - START_BLOCK
     environment:
       GRAPHNODE_URL:
-        from_secret: graphnode-url
+        from_secret: graphnode-url-staging
       IPFS_URL:
-        from_secret: ipfs-url
-      ENV: prod
+        from_secret: ipfs-url-staging
+      ENV: staging
     commands:
       - cd packages/subgraph
-      - export SUBGRAPH_NAME=bellecour/dev-dataprotector-v2
+      - export SUBGRAPH_NAME=bellecour/staging-dataprotector-v2
       - echo "deploying commit $DRONE_COMMIT on subgraph $SUBGRAPH_NAME"
-      - npm ci
       - npm run codegen
       - npm run build
       - npx graph create --node $GRAPHNODE_URL $SUBGRAPH_NAME
-      - npx graph deploy --node $GRAPHNODE_URL $SUBGRAPH_NAME --ipfs $IPFS_URL --version-label $DRONE_COMMIT
+      - npx graph deploy --node $GRAPHNODE_URL $SUBGRAPH_NAME --ipfs $IPFS_URL --version-label $(npm pkg get version)
```

Comment:
I see that you use both `$(npm pkg get version)` and `$DRONE_COMMIT` in different places. Why not use only one everywhere ?